src/py/mains/run.py

```diff
@@ -87,6 +89,9 @@ def wait_for_file(filename, task, job_name = ""):
             if support.job_type == "SGE":
                 fc_run_logger.info( "deleting the job by `qdel` now..." )
                 os.system("qdel %s" % job_name) # Failure is ok.
+            if support.job_type == "SLURM"
+            	fc_run_logger.info( "Deleting the job by 'scancel' now...")
```

Comment:
Do not mix spaces with tabs.  What crappy editor are you using? Serious question. Don't be embarrassed. I want to know so that I can tell people what settings to use.

This was also a common problem at Amazon, mostly caused by Eclipse, but the more sophisticated teams included an Eclipse format file with their Java source-code. In Go-lang, people simply run **goformat**, which is a wonderful tool! All languages should include an auto-formatter.

>  Unfortunately, tabs ARE considered equivalent to spaces -- specifically, eight spaces, as the docs say quite clearly. Never mixing tabs and spaces is best-practice, but you need to run Python with -t (or -tt) to get warnings (or errors) for violations of said best practice. –  Alex Martelli Jan 9 '10 at 19:12

I plan to add `-t` (and eventually `-tt`) to catch this sooner.